recipes/pcaone/build.sh

```diff
@@ -5,5 +5,10 @@ mkdir -p ${PREFIX}/bin
 export CFLAGS="-I$PREFIX/include"
 export CXXFLAGS="-I$PREFIX/include"
 export LDFLAGS="-L$PREFIX/lib"
-make MKLROOT=${PREFIX}
+axv=1
+if [ $(uname -s) == "Linux" ];then
+  grep "avx2" /proc/cpuinfo |grep "fma" &>/dev/null
```

Comment:
This will be executed on the build node and not in the user environment. So it can happen that you compile here against an instruction set that is not available on the user system I think.

Reply:
oh, thanks, good to know this won't work! any solution to the problem recommended?

Reply:
What bwa-mem2 (https://github.com/bwa-mem2/bwa-mem2/) does is have a precompiled binary for different instruction sets, and the actual bwa-mem2 binary selects the appropriate one for the user at runtime..

Reply:
thanks. what if I compile two versions then write a bash wrapper to detect the instruction sets at runtime to point to the right version? will it be a solution?

Reply:
Looks good?

Reply:
@cbrueffer Hi Christian, I updated the build.sh. Could you help review it?

Reply:
Looks like that should work.